### ISSUE TITLE
Installed 2.6.3/3019 manually - lost font personalizations

### ISSUE THREAD

**Issue by dajare** [mdbergmann/Eloquent#50]
Hi - When I launched Eloquent on OSX (Early 2015, Yosemite, 10.10.5), I got the pop-up saying a new version was available (2.3.6), but that the update mechanism was borked 😕 and to download and update manually. I did.

Unfortunately, when I then launched the freshly updated Eloquent, the last "state" was preserved, by my personalizations -- by which I mean font choices in particular -- were gone. Not a big problem, but if there was a way of making such optional choices "stick" across upgrades, that would of course be a wonderful thing.

**Comment by mdbergmann** [comment]
Any feedback on my question?

**Comment by dajare** [comment]
Sorry not to get back to you! I checked initially to see if I had that information, and had neither the original download, nor could I work out how to find which version I updated from.

But with this welcome prompt :) I just did a websearch, and found that I could get this information from **System Information > Software > Installations**. It seems that I first installed Eloquent on 09.07.2015, and it was version `2.4.8-1`. My new version is `2.6.3/3019`.

I hope that helps, and thanks for your interest.
